egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -28,6 +28,8 @@ def ParseSpliceString(splice_indexes, label_delay=None):
     if label_delay is not None:
         left_context = -label_delay
         right_context = label_delay
+    if preserve_state == "true":
```

Comment:
left_context should take the max of left_context among all outputs, including the state outputs. Unlike the ordinary output, the state outputs are not "label delayed", which means their label_delay=0. So in this case left_context=max(0, -label_delay)